utils/build-script-impl

```diff
@@ -2394,7 +2399,9 @@ for host in "${ALL_HOSTS[@]}"; do
                 call "${CMAKE_BUILD[@]}" "${build_dir}" $(cmake_config_opt ${product})
             fi
 
-            call "${CMAKE_BUILD[@]}" "${build_dir}" $(cmake_config_opt ${product}) -- "${BUILD_ARGS[@]}" ${build_targets[@]}
+            if [[ ${#build_targets[@]} -gt 0 ]] ; then
```

Comment:
Are you sure about this? Ninja builds all targets if no target is passed in, but I see that `libcxx` sets this to empty above. It might not build after this change.